githubbot/githubbot/db.go

```diff
@@ -28,6 +30,19 @@ func (d *DB) CreateSubscription(convID chat1.ConvIDStr, repo string, branch stri
 			VALUES
 			(?, ?, ?, ?, ?)
```

Comment:
you can do `ON DUPLICATE KEY UPDATE` like here https://github.com/keybase/managed-bots/blob/master/base/db.go#L69-L73

Reply:
oh neat!